pkg/hub/submarinerbrokerinfo/brokerinfo.go

```diff
@@ -70,9 +72,9 @@ type SubmarinerBrokerInfo struct {
 	CatalogStartingCSV        string
 	SubmarinerGatewayImage    string
 	SubmarinerRouteAgentImage string
+	SubmarinerGlobalnetImage  string
 	LighthouseAgentImage      string
 	LighthouseCoreDNSImage    string
-	LoadBalancerEnabled       bool
```

Comment:
Move this close to `NATEnabled` to fix the `golangci-lint` field alignment error.